client/src/components/PostDetails.jsx

```diff
@@ -17,6 +19,17 @@ export default function PostDetails() {
         getAndResetPost();
     }, []);
 
+    const handleSubscribeClick = () => {
+        const newSubscription = 
+        {
+            subscriberId: post.author?.id,
+            followerId: user.id
+        }
+        console.log(user)
```

Comment:
If this console log was for testing purposes and you don't need it anymore, it'd be appreciated if it was removed.